pyeudiw/sd_jwt/__init__.py

```diff
@@ -105,12 +105,29 @@ def import_pyca_pri_rsa(key, **params):
     )
     return jwcrypto.jwk.JWK(**params)
 
+def import_ec(key, **params):
+    pn = key.private_numbers()
+    params.update(
+        kty="EC",
+        crv="P-256",
```

Comment:
don't use constants but configurable parameters

here we're forcing P-256 within the code, while we must take it from the args and we may have a default fallback value